processing/src/main/java/io/druid/query/BaseQuery.java

```diff
@@ -62,6 +62,11 @@
     return parseBoolean(query, "finalize", defaultValue);
   }
 
+  public static <T> int getContextPopulateUncoveredIntervals(Query<T> query, int defaultValue)
```

Comment:
ah I see. this is a limit not a boolean. can this be `uncoveredIntervalsLimit` or similar?